build.gradle

```diff
@@ -519,12 +520,32 @@ forbiddenApisTest.enabled = false
  * Maven Publish *
  *****************/
 
+void addManifest(Jar jar) {
+    jar.manifest {
+        attributes 'Implementation-Title': project.name
+        attributes 'Implementation-Vendor': 'HiveMQ'
+        attributes 'Implementation-Version': project.version
```

Comment:
We already have a manifest for the jar and the contents are different (e.g. title is HiveMQ)
We should use something like
```
tasks.withType(Jar) {
    manifest {
        attributes ...
    }
}
```